Dockerfile

```diff
@@ -38,6 +38,7 @@ ENV OPENVPN_USERNAME=**None** \
     TRANSMISSION_DOWNLOAD_DIR=/data/completed \
     TRANSMISSION_INCOMPLETE_DIR=/data/incomplete \
     TRANSMISSION_WATCH_DIR=/data/watch \
+    TRANSMISSION_UMASK=2 \
```

Comment:
Is it needed? I'm nitpicking but:

The default transmission `umask` is already defined in the `settings.json`, which is modified if needed to respect environment variables **before** `userSetup.sh` is run. So we should use this instead IMO. For example with something like:

```
TRANSMISSION_UMASK=$(python3 -c 'import json; f = open("/config/settings.json"); settings = json.load(f); f.close(); print(settings["umask"])')
```

Reply:
(nitpicking on the nitpicking ;) )
`jq` is already available on the image, you can achieve the same with simpler code:
```
# jq .umask /config/settings.json
2
```